crawl-ref/source/dat/strings/de/annotations.txt

```diff
@@ -435,16 +435,16 @@ Gegenstand in falscher Position
 " (rechte Pfote)"
 %%%%
 " (left claw)"
-" (linke Kralle)"
+" (linke Klaue)"
 %%%%
 " (right claw)"
-" (rechte Kralle)"
+" (rechte Klaue)"
 %%%%
 " (left tentacle)"
-" (linker Tentakel)"
+" (linkes Tentakel)"
```

Comment:
Duden say der oder das Tentakel: https://www.duden.de/rechtschreibung/Tentakel. I also did a Google Ngram search, and masculine seems to be more common.

But if you still think das Tentakel is better, I will accept your call as the native speaker. We will have to update all the other places where Tentakel appears.

Reply:
Oh dear. There are some words Germans just can't agree on. This might be one of those. Der Tentakel sounds totally wrong to me. I've always used das Tentakel. But other native speakers might completely and strongly disagree. Let's stick with das, and if this project gains some traction and others disagree, we can always change it again.

Reply:
OK